guides/shipments/split-shipments.md

```diff
@@ -0,0 +1,108 @@
+# Split shipments
+
+Solidus supports split shipments out of the box. This feature addresses the
+needs of complex Solidus stores who require detailed inventory management, and
+sophisticated shipping and warehouse logic. It also allows you to manage
+shipments from multiple stock locations.
+
+[`Spree::Stock::SimpleCoordinator`][simple-coordinator] contains all of the
+business logic for how stock should be packaged. If your store requires a
+specialized flow for handling split shipments, the simple coordinator should
+provide a good starting point for customizations. 
+
+## Creating proposed shipments
+
+An order's shipments are determined by
+[`Spree::Stock::SimpleCoordinator`][simple-coordinator] while the 
+`Spree::Order`'s' state is set to `delivery`. This occurs before the customer
+has completed their order at checkout.
+
+The simple coordinator deletes any existing shipments for an order and then
+creates all of the shipments currently available for this order.
+
+The simple coordinator performs a number of tasks in order to create shipment
+proposals:
+
+1. The coordinator calculates the availability of the ordered items.
+2. Inventory is allocated from available stock to the current order.
+3. It splits the order into logical packages based on stock locations and
+   inventory at those locations.
+4. Each package generates a new `Spree::Shipment` object that is associated with
+   the current order.
+5. It estimates the shipping rates for each shipment.
+ 
+After the proposed shipments have been determined, the customer can continue the
+checkout process and take the order from the `delivery` state to the `payment`
+state.
```

Comment:
Maybe add a sentence here about how Solidus < 2.4 have a different legacy stock coordinator and associated classes that are more complex.

Reply:
Just added a note about this in [753c5ce](https://github.com/solidusio/solidus/pull/2345/commits/753c5cef3f08ef42e57243a7ad6595672d5f9058#diff-d78114ce442487a56a412a7c1241394eR38). Thanks.